src/test/FileUploader.spec.ts

```diff
@@ -86,3 +10,187 @@ const sampleResponse = `{
   "SYSJOBID" : "117382",
   "SYSWARNINGTEXT" : ""
 }`
+
+describe('FileUploader', () => {
+  let originalFetch: any
+
+  beforeAll(() => {
+    originalFetch = (global as any).fetch
+  })
+
+  beforeEach(() => {
+    ;(global as any).fetch = jest.fn().mockImplementation(() =>
+      Promise.resolve({
+        text: () => Promise.resolve(sampleResponse)
+      })
+    )
+  })
+
+  afterAll(() => {
+    ;(global as any).fetch = originalFetch
+  })
+
+  it('should upload successfully', async (done) => {
+    const fileUploader = new FileUploader(
```

Comment:
We can move `FileUploader` instance to the test suite scope and reuse it in every test.

Reply:
Done!